chia/wallet/puzzles/dao_proposal_validator.clsp

```diff
@@ -42,70 +36,73 @@
   )
 
   (defun-inline calculate_proposal_puzzle (
-      PROPOSAL_MOD_HASH
-      PROPOSAL_SINGLETON_STRUCT
-      PROPOSAL_TIMER_MOD_HASH
-      CAT_MOD_HASH
-      DAO_FINISHED_STATE_MOD_HASH
-      TREASURY_MOD_HASH
-      LOCKUP_MOD_HASH
-      CAT_TAIL_HASH
-      TREASURY_ID
+      PROPOSAL_SELF_HASH
+      proposal_singleton_id
       proposal_yes_votes
       proposal_total_votes
       proposal_innerpuz_hash
     )
-    (puzzle-hash-of-curried-function PROPOSAL_MOD_HASH
-      (sha256 ONE proposal_innerpuz_hash)
+    (puzzle-hash-of-curried-function PROPOSAL_SELF_HASH
       (sha256 ONE proposal_total_votes)
       (sha256 ONE proposal_yes_votes)
-      (sha256 ONE TREASURY_ID)
-      (sha256 ONE CAT_TAIL_HASH)
-      (sha256 ONE LOCKUP_MOD_HASH)
-      (sha256 ONE TREASURY_MOD_HASH)
-      (sha256 ONE DAO_FINISHED_STATE_MOD_HASH)
-      (sha256 ONE CAT_MOD_HASH)
-      (sha256 ONE PROPOSAL_TIMER_MOD_HASH)
-      (sha256 ONE PROPOSAL_MOD_HASH)
-      (sha256tree PROPOSAL_SINGLETON_STRUCT)
+      (sha256 ONE proposal_innerpuz_hash)
+      (sha256 ONE proposal_singleton_id)
+      (sha256 ONE PROPOSAL_SELF_HASH)
     )
   )
 
-  (assert
+  (if
```

Comment:
Again, this is a big readability concern, and I think assert does this as a macro

Reply:
leaving it as if was a mistake, heldover from debugging fixed now